lib/interfaces/heroku_interface.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../http_post_interface'
+Dir['./lib/slack_commands/*.rb'].each { |file| require file }
+
+class HerokuInterface < HttpPostInterface
+  def handle_heroku_webhook(payload)
+    params = Rack::Utils.parse_nested_query(payload)
+    user = params['user'].split('@')[0]
+    release = "#{params['release']} " if params['release']
+    server = params['url'].split('/').last.split('.').first[8..-1]
+
+    deploys.complete_deploy(server)
+
+    SlackCommand
+      .new
+      .send_to_slack("#{user} just finished deploying #{release}to #{server}.")
```

Comment:
Not wild about this pattern (instantiating a SlackCommand just to call send_to_slack).  I'd argue it's probably worth moving the `send_to_slack` function somewhere else- maybe SlackInterface.  Or probably more appropriately, a new SlackApi class which would fit the existing pattern (XInterface for incoming call from X, XApi for outgoing calls to X).